src/Microsoft.AspNet.Mvc.Xml/project.json

```diff
@@ -0,0 +1,25 @@
+﻿{
+    "description": "Contains DataContractSerializer and XmlSerializer based input and output formatters",
+    "version": "6.0.0-*",
+    "compilationOptions": {
+        "warningsAsErrors": false
```

Comment:
ah yes, somehow i missed it...I copied it from WebApiCompatShim project actually...